spec/javascripts/components/story/expanded_story/expanded_story_default/expanded_story_default_spec.js

```diff
@@ -1,77 +1,165 @@
-import React from 'react';
-import { shallow } from 'enzyme';
 import { ExpandedStoryDefault } from 'components/story/ExpandedStory/ExpandedStoryDefault';
-import storyFactory from '../../../../support/factories/storyFactory';
+import React from 'react';
 import { createTemporaryId } from '../../../../../../app/assets/javascripts/models/beta/story';
+import storyFactory from '../../../../support/factories/storyFactory';
+import { renderWithProviders } from '../../../setupRedux';
 
 describe('<ExpandedStoryDefault />', () => {
-  const defaultProps = () => ({
-    story: {
-      ...storyFactory(),
-      _editing: storyFactory(),
-    },
-    onEdit: vi.fn(),
-    storyFailure: vi.fn(),
-    createTask: vi.fn(),
-    deleteTask: vi.fn(),
-    toggleTask: vi.fn(),
-    deleteNote: vi.fn(),
-    createNote: vi.fn(),
-    addLabel: vi.fn(),
-    removeLabel: vi.fn(),
-    setLoadingStory: vi.fn(),
-    users: [],
-    project: { labels: [] },
-    enabled: true,
-    onClone: vi.fn(),
-    showHistory: vi.fn(),
-    disabled: false,
+  beforeEach(function () {
```

Comment:
Perhaps we can remove this line. It doesn't do nothing.

Reply:
If `vi` isn't spying it cannot expect calls, so removing the setup breaks tests

![image](https://github.com/user-attachments/assets/f37b3ffd-4026-47ff-9db5-f0170eb13218)